dns-controller/README.md

```diff
@@ -10,13 +10,12 @@ we just expose it via DNS.
 
 The dns-controller recognizes annotations on nodes.
 
-`dns.alpha.kubernetes.io/external` will set up records for accessing the resource externally
-
-`dns.alpha.kubernetes.io/internal` will set up records for accessing the resource internally
+* `dns.alpha.kubernetes.io/external` will set up records for accessing the resource externally
+* `dns.alpha.kubernetes.io/internal` will set up records for accessing the resource internally
 
 When added on `Service` controllers:
 
-`dns.alpha.kubernetes.io/external` creates a Route53 A record with `public` IPs of all the nodes
-`dns.alpha.kubernetes.io/internal` creates a Route53 A record with `private` IPs of all the nodes
+* `dns.alpha.kubernetes.io/external` creates a Route53 A record with the IPs of the load balancer (type=LoadBalancer) or all the nodes (type=NodePort)
+* `dns.alpha.kubernetes.io/internal` creates a Route53 A record with spec.clusterIP of the service
```

Comment:
We're experimenting with using Kubernetes's internal network as company's VPN. I'm currently pushing Kubernetes's DNS server as part of the VPN configuration, but standard domain names are not too friendly (gitlab.gitlab.svc.cluster.local for example), so I'm exploring alternatives for that.